### PR TITLE
Update FAQ

### PR DESCRIPTION
**Pull request by douglas-reid**
Based on FAQ doc in Team Drive. With this PR, the team drive doc will then be marked as obsolete.

### CODE REVIEW

File: faq/index.md [hunk 1]
```diff
@@ -10,31 +10,101 @@ type: markdown
...
+*   [Manager](https://github.com/istio/manager/tree/master/doc)
+*   [Mixer](https://github.com/istio/mixer/tree/master/doc)
+
+#### Istio doesn't work - what do I do?
+
+Follow the [instructions](https://github.com/istio/istio/blob/master/CONTRIBUTING.md#issues) to open an issue [here](https://github.com/istio/istio/issues/new) or ask questions on [slack](https://istio-dev.slack.com/messages/C524NCGR1/). Our [users mailing list](https://groups.google.com/forum/#!forum/istio-users) is another great way to get help and answers. We also monitor [Stack Overflow](https://stackoverflow.com/questions/tagged/istio) for questions tagged with "istio".
+
+Additionally, we provide [Reference Guides](https://istio.io/docs/reference/) for all of the Istio components. These can be helpful when troubleshooting issues with configuration, etc.
+
+#### Why Alpha? And what does the Alpha release cover?
```
Comment (rshriram) [inline faq/index.md]: del Why Alpha and just say "What features are provided by the Alpha release?"

File: faq/index.md [hunk 2]
```diff
@@ -10,31 +10,101 @@ type: markdown
...
+*   L7 traffic routing rules
+*   In-cluster load balancing for HTTP, GRPC & TCP
+*   Cluster Ingress and Egress
+*   Fault injection
+*   In-memory rate limiting
+*   L7 Metrics and Logs collection
+*   Secure service-to-service authentication with strong identity
+*   Pluggable policy layer and configuration API
+
+#### Does Istio support authn/authz based on attributes in addition to service account (e.g. [Kubernetes Network Policy](https://kubernetes.io/docs/concepts/services-networking/networkpolicies/) based on labels)?
```
Comment (rshriram) [inline faq/index.md]: no abbreviation. Use full forms?
Second, this kind of jumps out of the blue and assumes knowledge of attributes, service accounts, etc.
Can't this whole question be simplified to Does Istio Auth support Authorization? (Eventually yes, but not currently). The bit about mTLS and labels should go into concepts about Istio auth, shouldn't it?

File: faq/index.md (hunk 1 above)
Comment (douglas-reid) [inline faq/index.md]: Done.

File: faq/index.md (hunk 2 above)
Comment (douglas-reid) [inline faq/index.md]: Updated in attempt to match comments. PTAL.

**Comment by douglas-reid** [comment]
PTAL. Updated to refine some of the auth sections.

File: faq/index.md [hunk 3]
```diff
@@ -10,31 +10,99 @@ type: markdown
...
+
+*   [Envoy](https://lyft.github.io/envoy/docs/)
+*   [Manager](https://github.com/istio/manager/tree/master/doc)
+*   [Mixer](https://github.com/istio/mixer/tree/master/doc)
+
+#### Istio doesn't work - what do I do?
+
+Follow the [instructions](https://github.com/istio/istio/blob/master/CONTRIBUTING.md#issues) to open an issue [here](https://github.com/istio/istio/issues/new) or ask questions on [slack](https://istio-dev.slack.com/messages/C524NCGR1/). Our [users mailing list](https://groups.google.com/forum/#!forum/istio-users) is another great way to get help and answers. We also monitor [Stack Overflow](https://stackoverflow.com/questions/tagged/istio) for questions tagged with "istio".
+
+Additionally, we provide [Reference Guides](https://istio.io/docs/reference/) for all of the Istio components. These can be helpful when troubleshooting issues with configuration, etc.
```
Comment (geeknoid) [inline faq/index.md]: (/docs/reference)

File: faq/index.md [hunk 4]
```diff
@@ -10,31 +10,99 @@ type: markdown
...
+#### What does the Alpha release cover?
+
+The Alpha release of Istio provides an early preview of the intended functionality and user experience of our service mesh. We are hoping to solicit early feedback on direction and design decisions.
+
+The Alpha release includes the following features:
+
+*   Simple command-line installation into a kubernetes cluster
+*   Scripted proxy injection with traffic capture via iptables
+*   L7 traffic routing rules
+*   In-cluster load balancing for HTTP, GRPC & TCP
```
Comment (geeknoid) [inline faq/index.md]: gRPC

File: faq/index.md [hunk 5]
```diff
@@ -10,31 +10,99 @@ type: markdown
...
+
+Additionally, we provide [Reference Guides](https://istio.io/docs/reference/) for all of the Istio components. These can be helpful when troubleshooting issues with configuration, etc.
+
+#### What does the Alpha release cover?
+
+The Alpha release of Istio provides an early preview of the intended functionality and user experience of our service mesh. We are hoping to solicit early feedback on direction and design decisions.
+
+The Alpha release includes the following features:
+
+*   Simple command-line installation into a kubernetes cluster
```
Comment (geeknoid) [inline faq/index.md]: But a blank line between each bullet, it spaces the HTML better.

File: faq/index.md [hunk 6]
```diff
@@ -10,31 +10,99 @@ type: markdown
...
+
+#### Does Istio Auth use kubernetes secrets?
+
+Yes. The key and certificate distribution in Istio Auth is based on [kubernetes secrets](https://kubernetes.io/docs/concepts/configuration/secret/).
+
+Secrets have known [security risks](https://kubernetes.io/docs/concepts/configuration/secret/#risks). The kubernetes team is working on [several features](https://docs.google.com/document/d/1T2y-9geg9EfHHtCDYTXptCa-F4kQ0RyiH-c_M1SyD0s) to improve kubernetes secret security, from secret encryption to node-level access control. And as of version 1.6, kubernetes introduces [RBAC authorization](https://kubernetes.io/docs/admin/authorization/rbac/), which can provide fine-grained secrets management.
+
+#### What kind of traffic does Istio Auth support?
+
+For Alpha, we only support http traffic. And we are actively working on supporting more kinds of traffic like SQL, etc.
```
Comment (geeknoid) [inline faq/index.md]: http -> HTTP
Or should that be HTTPS?

File: faq/index.md [hunk 7]
```diff
@@ -10,31 +10,99 @@ type: markdown
...
+
+#### What is the roadmap for Istio?
```
Comment (geeknoid) [inline faq/index.md]: What is Istio's roadmap?

File: faq/index.md (hunk 5 above)
Comment (douglas-reid) [inline faq/index.md]: done.

File: faq/index.md [hunk 8]
```diff
@@ -10,31 +10,99 @@ type: markdown
...
+Traditionally, much of the logic handled by Istio has been built directly into applications. Across a fleet of services, managing updates to this communications logic can be a large burden. Istio provides an infrastructure-level solution to managing service communications.
+
+*Application developers*: With Istio managing how traffic flows across their services, developers can focus exclusively on business logic and iterate quickly on new features.
+
+*Service operators*: Istio enables policy enforcement and mesh monitoring from a single centralized control point, independent of application evolution. As a result, operators can ensure continuous policy compliance through a simplified management plane.
 
 #### How do I get started using Istio?
 
-TBD
+We recommend starting with the [BookInfo sample](https://istio.io/docs/samples/bookinfo.html). The BookInfo example walks through setting up a cluster with four distinct microservices managed by Istio. It exercises some basic features, including content-based routing, fault injection, and rate-limiting.
```
Comment (douglas-reid) [inline faq/index.md]: done.

File: faq/index.md [hunk 9]
```diff
@@ -10,31 +10,99 @@ type: markdown
...
+
+After you have mastered the BookInfo sample, you are ready to begin using Istio for your own services. To start using Istio on your existing Kubernetes cluster, please refer to our [Installation](https://istio.io/docs/tasks/istio-installation.html) task guide.
```
Comment (douglas-reid) [inline faq/index.md]: done

File: faq/index.md (hunk 4 above)
Comment (douglas-reid) [inline faq/index.md]: done.

File: faq/index.md (hunk 6 above)
Comment (douglas-reid) [inline faq/index.md]: Made it HTTP. Assuming HTTP, as input was direct from auth team. They would have to update if it should be HTTP(S).

File: faq/index.md (hunk 7 above)
Comment (douglas-reid) [inline faq/index.md]: done.

**Comment by douglas-reid** [comment]
Thanks for the review!

**Pull request merged**